src/components/Filters/FilterByName/FilterByName.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components/macro'
+
+export default function FilterByName({ userInputName, setUserInputName }) {
+  return (
+    <FilterLabel>
+      <h2>Name:</h2>
+      <input
+        placeholder="z.B. Minikleid"
+        value={userInputName}
+        onChange={event => setUserInputName(event.target.value)}
+      />
+    </FilterLabel>
+  )
+}
+
+const FilterLabel = styled.label`
```

Comment:
It looks like you have a lot of same code/styles in your filter components. For a refactor in the future you could think about how to reduce repetition.